Assets/Scripts/EOSSessionsManager.cs

```diff
@@ -1862,5 +2021,419 @@ public void DeclineLobbyInvite()
+            // First find a local session with this name
+            Session localSession = GetSession(localSessionName);
+
+            if (localSession == null)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (InformSessionMembersToRefresh): No local session with name \"{0}\" was found.", localSessionName);
+                return;
+            }
+
+            ActiveSession activeSession = localSession.ActiveSession;
+
+            if (activeSession == null)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (InformSessionMembersToRefresh): Found local session with name \"{0}\", but there was no corresponding ActiveSession. An ActiveSession should be populated in InitActiveSession.", localSessionName);
+                return;
+            }
+
+            // Copy over the session information
+            // This should contain SessionDetails, which will include the user id of the owner of the room 
+            ActiveSessionCopyInfoOptions copyInfoOptions = new ActiveSessionCopyInfoOptions() { };
+            ActiveSessionInfo? copiedInfo;
+            Result activeSessionInfoCopyResult = activeSession.CopyInfo(ref copyInfoOptions, out copiedInfo);
+
+            if (activeSessionInfoCopyResult != Result.Success)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (InformSessionMembersToRefresh): Found local and active session with name \"{0}\", but failed to copy its info. Result code {1}", localSessionName, activeSessionInfoCopyResult);
+                return;
+            }
+
+            if (!copiedInfo.Value.SessionDetails.HasValue)
+            {
+                Debug.LogErrorFormat("EOSSessionsManager (InformSessionMembersToRefresh): Found local and active session with name \"{0}\", but the SessionDetails in the object is null. Cannot determine session id.", localSessionName, activeSessionInfoCopyResult);
+                return;
+            }
+
+            // Are we the owner of this session? We should only be messaging people if we are, inside this function
+            if (!copiedInfo.Value.SessionDetails.Value.OwnerUserId.Equals(EOSManager.Instance.GetProductUserId()))
+            {
+                // We're not the owner! This is not an error state, but don't send any messages.
+                return;
+            }
+
+            ActiveSessionGetRegisteredPlayerCountOptions countOptions = new ActiveSessionGetRegisteredPlayerCountOptions() { };
+            uint playerCount = activeSession.GetRegisteredPlayerCount(ref countOptions);
+
+            Log($"EOSSessionsManager (InformSessionMembers): There are {playerCount} registered members in {localSession.Name}, informing users of {messageDetail} (excluding self)");
+
+            string messageToSend = string.Format(P2PInformSessionMessageFormat, copiedInfo.Value.SessionDetails.Value.SessionId, EOSManager.Instance.GetProductUserId(), messageDetail);
+            for (uint ii = 0; ii < playerCount; ii++)
```

Comment:
I recall you mentioning sometime when we were discussing things in-person that you had a reason for this - but I find the use of `ii` to be an odd choice for a loop variable. Could you elaborate on your thought process for doing so?

Reply:
Compared to `i`, `ii` is less frequent in code. That means if I use `ii` as an iterator, I can search for `ii` using `CTRL+F` and see more places where it's used. If I use `i`, then searching the text for `i` will get more hits that I don't care about.

It doesn't _stand_ for anything, like there's no acronym or abbreviation here. I just find `ii` more clear when trying to read code than using `i`.